manifests/4.10/supported-nic-ids_v1_configmap.yaml

```diff
@@ -22,8 +22,6 @@ data:
   Nvidia_mlx5_ConnectX-5_Ex: 15b3 1019 101a
   Nvidia_mlx5_ConnectX-6: 15b3 101b 101c
   Nvidia_mlx5_ConnectX-6_Dx: 15b3 101d 101e
-  Nvidia_mlx5_ConnectX-6_Lx: 15b3 101f 101e
```

Comment:
Why removing these two?

Reply:
It was synced from upstream. I think we shall keep it unchanged.

Reply:
fixed